packages/matter.js/src/behavior/supervision/RootSupervisor.ts

```diff
@@ -110,7 +110,11 @@ export class RootSupervisor implements ValueSupervisor {
         if (!persistent) {
             persistent = new Set();
             for (const member of this.#members) {
-                if (member.effectiveQuality.nonvolatile) {
+                if (
+                    member.effectiveQuality.nonvolatile ||
+                    member.effectiveAccess.writable ||
+                    member.effectiveAccess.fabricScoped
```

Comment:
Spec tells that non volatile needs to be persisted but semantically other attributes could also be. So I decided back then that a writable attribute (mostly config stuff or behavior changing) should also be persisted because else a written change is gone on next reboot which makes no sense. Ok fabric scoped is sameish but here writable is mostly there too. So it is "interpreting specs topic". I can later post official spec phrase for persisting for that too.

Reply:
https://github.com/orgs/project-chip/projects/11/views/1?pane=issue&itemId=64811050 and added TODO